javascript-light-bulb/main.js

```diff
@@ -1,16 +1,10 @@
-var clicks = 0;
-
 function lightSwitch(event) {
-  clicks++;
-  if (clicks === 1) {
+  if (button.className === 'light on') {
```

Comment:
This is reliant on the class name on the DOM, which makes your JavaScripot dependent on the state of the DOM, which is potentially hazardous. You need a variable that holds a boolean value that matches the current state of the lightbulb on the page. `var isLightOn = false` if the HTML starts with the light off, for example.